Mlem/Views/Tabs/Settings/Components/SettingsView.swift

```diff
@@ -38,11 +39,11 @@ struct SettingsView: View {
                                 )
                                 .padding(.vertical, -6)
                                 .padding(.leading, 3)
-                                if let account = appState.currentActiveAccount {
+                                if let person = siteInformation.myUserInfo?.localUserView.person {
                                     VStack(alignment: .leading, spacing: 3) {
-                                        Text(account.nickname)
+                                        Text(appState.currentActiveAccount?.nickname ?? info.localUserView.person.name)
```

Comment:
Build fails here--maybe something like

```suggestion
                                        Text(appState.currentActiveAccount?.nickname ?? person.name)
```